install/migrations/update_10.0.x_to_11.0.0/form.php

```diff
@@ -117,7 +118,8 @@
             KEY `date_creation` (`date_creation`),
             KEY `date_mod` (`date_mod`),
             KEY `forms_forms_id` (`forms_forms_id`),
-            KEY `users_id` (`users_id`)
+            KEY `users_id` (`users_id`),
+            KEY `entities_id` (`entities_id`)
```

Comment:
For some reasons, answers need to have an entity to allow the new search option (find ticket per form answer).
It make sense for answers to be linked to an entity, so not really a problem.

I didn't include `is_recursive` to be keep the same behavior as tickets ( = never recursive).